Formula/broot.rb

```diff
@@ -19,6 +20,24 @@ class Broot < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+
+    # Replace man page "#version" and "#date" based on logic in release.sh
+    build_date = Utils.safe_popen_read("date -u +'%Y/%m/%d' 2> /dev/null").chomp
+    inreplace "man/page" do |s|
+      s.gsub! "#version", version
+      s.gsub! "#date", build_date
```

Comment:
```suggestion
    inreplace "man/page" do |s|
      s.gsub! "#version", version
      s.gsub! "#date", Time.now.strftime("%Y/%m/%d")
```
We don't need to shell out to `date` here.

Reply:
Sounds good. I was mainly basing on how some other formulas were doing it. I'll try modifying and rerunning.

Is there a preference of running using local time or UTC (`Time.now.utc.strftime("%Y/%m/%d")`)?

Reply:
Not really; not sure it matters that much. Use whichever you prefer.